SpiceSharpTest/Models/Framework.cs

```diff
@@ -73,13 +73,13 @@ protected void TestAC(Netlist netlist, double[] reference)
                 // Test real part
                 double actual = netlist.Exports[0].Extract(data);
                 double expected = reference[index++];
-                double tol = Math.Max(Math.Abs(actual), Math.Abs(expected)) * 1e-6 + 1e-30;
+                double tol = Math.Max(Math.Abs(actual), Math.Abs(expected)) * 1e-6 + 1e-12;
```

Comment:
Could you please tell me why 1e-30 => 1e-12 ?

Reply:
I believe numerical errors caused some tests to fail so I relaxed the tolerance. I will check this again to give a more detailed analysis of why it fails at 1e-30.

Reply:
After some investigation it is for a big part because of issue #7. There are still some leftover differences that I will look into before doing applying the request.